networking/src/Ntp/Client.hs

```diff
@@ -42,10 +42,13 @@ import           Ntp.Util (createAndBindSock, resolveNtpHost, selectIPv4, select
                            udpLocalAddresses, withSocketsDoLifted)
 
 data NtpStatus =
+      -- | The difference between ntp time and local system time
       NtpDrift Microsecond
```

Comment:
Such that if you add it to the local time, then you get the NTP time?

Reply:
Yes